shared/src/packages/users/libs/enums/users-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 const UsersApiPath = {
   ROOT: '/',
+  FOLLOW: '/:id/follow',
```

Comment:
To keep it consistent with value
```suggestion
  $ID_FOLLOW: '/:id/follow',
```

Reply:
[5509f38](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/280/commits/5509f38433b30fe7a12f4abd7441e6c40deda542)